.github/workflows/download_cleanup.yml

```diff
@@ -25,10 +25,42 @@ jobs:
         if: env.SSH_AUTH_SOCK != null
         run: |
           mkdir empty_folder
-          echo build-checks-fix/ >> include_file.txt
-          echo build-checks-fix/* >> include_file.txt
-          rsync --verbose --archive --times --recursive --delete --include-from=include_file.txt --exclude=* "empty_folder/" "${SSH_USER}@${SSH_HOST}:${DESTDIR}/"
+          echo 1.12/ > include_file.txt
+          echo 1.12/* >> include_file.txt
+          echo 1.12/*/* >> include_file.txt
```

Comment:
why do we need all these sub and sub-sub includes?

Reply:
I have not managed to find a way to delete an non empty directory. This ways the sub directories are emptied and than the directory itself is removed.

Reply:
okay, let's try.

Reply:
So this kinda worked for the dependencies dir, but /snapshots/1.12 and 2.0 are still there because the have non-empty sub.sub-...-sub-dirs :laughing: 

How much free space is free now?